lib/shoryuken/environment_loader.rb

```diff
@@ -15,9 +15,12 @@ def initialize(options)
     end
 
     def load
-      initialize_options
-      initialize_logger
-      load_rails if options[:rails]
+      if options[:rails]
+        load_rails
+      else
+        initialize_options
+        initialize_logger
```

Comment:
> I want to use gems like SettingsLogic

@yujideveloper how would that let people to use `--rails` and load settings without using `SettingsLogic` or similar?